src/main/java/com/onarandombox/MultiverseCore/listeners/MVPortalListener.java

```diff
@@ -38,13 +37,13 @@ public MVPortalListener(MultiverseCore core) {
      * @param event The event where an entity created a portal.
      */
     @EventHandler
-    public void entityPortalCreate(EntityCreatePortalEvent event) {
+    public void entityPortalCreate(PortalCreateEvent event) {
         if (event.isCancelled() || event.getBlocks().size() == 0) {
             return;
         }
         MultiverseWorld world = this.plugin.getMVWorldManager().getMVWorld(event.getEntity().getWorld());
         // We have to do it like this due to a bug in 1.1-R3
-        if (world != null && !world.getAllowedPortals().isPortalAllowed(event.getPortalType())) {
+        if (world != null && !world.getAllowedPortals().isPortalAllowed(PortalType.CUSTOM)) {
```

Comment:
This seems incorrect to me.

Reply:
I did that because I was getting an error there.